cli/disputer.go

```diff
@@ -150,6 +154,10 @@ var disputerStartCmd = &cli.Command{
 
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
+			return IncorrectNumArgs(cctx)
+		}
+
```

Comment:
oh this is an interesting one! you've stumbled upon a command with very bad settings, let's be a bit more radical here:

1. nargs should be strictly `0`, it doesn't actually accept an unflagged argument at all
2. In `Flags` above, `"start-epoch"` should be renamed to `"height"` because that's what it actually reads for that 
property
3. `"max-fee"` needs to be listed in `Flags`, it's optional and could be described as `"maximum fees to spend on a dispute message"`
4. `"from"` needs to be lited in `Flags`, it's optional and could be described as `"the address to send dispute messages from if not the default wallet address"`

While you're in there, `"max-fee"` and `"from"` `Flags` can be copied as they are into `disputerMsgCmd` which also takes those flags but doesn't list them.